.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,74 @@
+name: CD
+
+on:
+  push:
+    branches: [main, ci-cd]
```

Comment:
I added a trigger for the `ci-cd` branch for testing purposes. If we need to add more stuff to the CI/CD pipeline next time, we can use this branch to test